package/python-kiwi-spec-template

```diff
@@ -113,6 +113,7 @@ Requires:       kiwi-tools
 Requires:       mtools
 Requires:       rsync
 Requires:       tar >= 1.2.7
+Requires:       openssl
```

Comment:
Not sure if OBS handles this well, but you could also require `/usr/bin/openssl` directly, if kiwi just needs the openssl binary. (IIRC OBS does not handle file provides very well though and not sure if this really simplifies things though…)

Reply:
I'd like to avoid that. obs doesn't handle file provides at all to my knowledge which was the reason in the past (and present) that some packages must be mentioned explicitly in the kiwi XML description because the obs worker and dependency solver did not pull them in as they are only a file provides.

Reply:
zypper can resolve them but obs doesn't as far as I know. We also need to make sure the dependency is correct for the other distributions and their package management system. I hope for Neal to check that part :)

Reply:
This is fine for Fedora and Debian/Ubuntu.